docker-compose/nginx/templates/ci.conf.template

```diff
@@ -0,0 +1,77 @@
+server {
+	listen      80;
+	listen      [::]:80;
+	server_name ${NGINX_ARTIFACTS_VHOST};
+	return      301 https://$server_name$request_uri;
+}
+
+# Build artifacts location
+server {
+	listen 443 ssl http2;
+	listen [::]:443 ssl http2;
+
+	server_name ${NGINX_ARTIFACTS_VHOST};
+
+	root /srv/buildbot/packages/;
+	location /helper_files {
+		alias /srv/buildbot/helper_files; #FIXME - for consistency, on hz-bbm2 let's rename it to helper_files instead of mariadb-shared-packages (current PROD)
+	}
+	location /galera {
+		alias /srv/buildbot/galera_packages;
+	}
+  #FIX ME - Still needed? Not present in DEV.
```

Comment:
Yes, but no, remove it and let's find another solution for cloud-init images, it's polluting and confusing.